.github/workflows/production-daily.yml

```diff
@@ -94,14 +94,14 @@ jobs:
 
       - name: Sentry query
         run: |
-          python __main__.py --report-type sentry-issues
-          python __main__.py --report-type sentry-rates
+          python __main__.py --report-type sentry-issues --project firefox-ios
+          python __main__.py --report-type sentry-rates --project firefox-ios
```

Comment:
The changes in `production-daily.yml` are necessary to keep the existing `firefox-ios` related queries going. Previously, the work only deals with `firefox-ios`.

Reply:
We will add the column before